python/README.md

```diff
@@ -1,38 +1,27 @@
 # nGraph python binding
 
-## Installation (If you haven't build nGraph yet.)
+## Installation
 
-Checkout nGraph and python wrapper code and build bdist wheel.
+### Install nGraph (Required)
 
 ```
-git clone https://github.com/NervanaSystems/ngraph.git
-cd ngraph/python
+Follow the [steps](http://ngraph.nervanasys.com/docs/latest/install.html) to build and install ngraph.
 ```
 
-```
-pip install wheel
-```
+Set the environment variables
 
-To build python2 bdist wheel type
 ```
-./build_python2_wheel.sh
-```
-To build python3 bdist wheel type
-```
-./build_python3_wheel.sh
+export NGRAPH_CPP_BUILD_PATH=$HOME/ngraph_dist/
+export LD_LIBRARY_PATH=$HOME/ngraph_dist/lib/
+export DYLD_LIBRARY_PATH=$HOME/ngraph_dist/lib (Only needed on MacOS)
```

Comment:
Would be better written as:
`export DYLD_LIBRARY_PATH=$HOME/ngraph_dist/lib # (Only needed on MacOS) `
The `#` prevents a problem if someone copy-pastes this whole block.